README.md

```diff
@@ -88,6 +113,28 @@ To deploy PromDash behind a reverse proxy you can set a global path prefix
 using the environment variable `PROMDASH_PATH_PREFIX`. Once set all URLs will
 start with the given prefix.
 
+One simple way to secure your Prometheus deployment with authentication is running a reverse proxy, Prometheus and PromDash on the same machine. The goal is to have a publicly available deployment for authenticated users via your reverse proxy while restricting raw `IP:PORT` access. We wont go cover any details on how to configure your reverse proxy, it depends on what you are using.
```

Comment:
I'm an Oxford comma fan, especially since it can make this listing less ambiguous :)

So "reverse proxy, Prometheus, and PromDash".